.docker/entrypoint/spec/jekyll_config_provider_spec.rb

```diff
@@ -43,12 +45,19 @@
     end
 
     context 'non-existing _config.yml' do
+      uuid = SecureRandom.urlsafe_base64
+      non_existing_directory = File.join(__dir__, 'data', ".#{uuid}")
+
       context 'serve returns config' do
         before(:all) do
-          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new('non_existing_directory')
+          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new(non_existing_directory)
           @jekyll_config = jekyll_config_provider.provide('serve')
         end
 
+        after(:all) do
+          FileUtils.rm_rf(non_existing_directory) if Dir.exist? non_existing_directory
```

Comment:
Codacy found an issue: [Modifier form of `if` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)